src/ExecuteSelectorCondition.sol

```diff
@@ -17,10 +17,15 @@ contract ExecuteSelectorCondition is
     IPermissionCondition,
     DaoAuthorizable
 {
+    /// @notice Contains a list of selectors for the given target (where) address
     struct SelectorTarget {
+        /// @notice The address where the selectors below can be invoked
         address where;
+        /// @notice The list of function selectors that can be invoked within an execute() call.
+        /// @notice Plain eth transfers should contain 0 as a selector.
```

Comment:
I wonder if it's better to set a magic number as the selector for ETH transfers

For example `0xeeeeeeee`

Reply:
We could do that, but it's as much arbitrary as setting 0x0 IMO

In fact, 0x0 signifies the absence of a selector, which is what happens when you only transfer eth